openlibrary/templates/account/create.html

```diff
@@ -3,7 +3,9 @@
 $putctx("cssfile", "signup")
 
 $ i18n_strings = {
-    $ "password_length_err": _("Must be between 3 and 20 characters")
+    $ "input_length_err": _("Must be between 3 and 20 characters"),
```

Comment:
Let's have two variables, `password_length_err` and `username_length_err`; even though their values will be the same. This is a little brittle if in the future we try to change the length restriction for one these; separate variables will avoid coupling this coincidentally same values.